packages/tradershub/src/components/StaticLink/StaticLink.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { ComponentProps } from 'react';
+import { Link, qtMerge } from '@deriv/quill-design';
+import { getStaticUrl } from '../../helpers/urls';
+
+type StaticLinkProps = {
+    children: React.ReactNode;
```

Comment:
Can we directly destructure ReactNode, AnchorHTMLAttributes

Reply:
can